extensions/cornerstone/src/Viewport/OHIFCornerstoneViewport.tsx

```diff
@@ -256,7 +230,9 @@ const OHIFCornerstoneViewport = React.memo(props => {
     setImageScrollBarHeight();
 
     return () => {
-      storePresentation();
+      commandsManager.runCommand('storePresentation', {
+        viewportIndex,
```

Comment:
Please use viewportId throughout - we are trying to remove references to viewportIndex generally.

Reply:
that is another big PR that I will care later